recipe/meta.yaml

```diff
@@ -11,21 +11,22 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
   skip: True  # [py<36]
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
-    - python
+    - python >=3.6
```

Comment:
Normally it's controlled by the `skip: True  # [py<36]` option (above) for the architecture-dependent packages.

Reply:
And it removed a half of the build configurations. I think it should be reverted to have builds for Python 3.6 and 3.7 for each platform (6 in total).